package exception_demo;

public class demo {

	public static void main(String[] args) {
	
		System.out.println("1");	
		System.out.println("2");
		System.out.println("3");
		System.out.println("4");
		
		
		System.out.println(10/0);
		//disturbing normal flow of exception
		
		System.out.println("5");
		System.out.println("6");
		System.out.println("7");
		System.out.println("8");
		System.out.println("9");
	}

}
